packages/dds/spanner/src/spannerHandle.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { SharedObject } from "@fluidframework/shared-object-base";
+import { Spanner } from "./spanner";
+
+/**
+ * Represents a handle to a Spanner object.
+ * TOld - The type of the old object.
+ * TNew - The type of the new object.
+ */
+export class SpannerHandle<TOld extends SharedObject, TNew extends SharedObject>
```

Comment:
Why do we need a new handle class?  I would expect the customer would just get the `Spanner` directly and then call APIs on it to retrieve the correct contained object.

Reply:
It totally should return the Spanner object, let me fix that